subsys/sd/mmc.c

```diff
@@ -233,7 +245,7 @@ static int mmc_send_op_cond(struct sd_card *card, int ocr)
 
 static int mmc_set_rca(struct sd_card *card)
 {
-	struct sdhc_command cmd;
+	struct sdhc_command cmd = {0};
```

Comment:
these changes to the `mmc_set_rca` function should be squashed into the first commit that introduces the function